### ISSUE TITLE
Minitest focus should be covered

### ISSUE THREAD

**Issue by KlotzAndrew** [rubocop-hq/rubocop#3773]
Debugging `focus` statements not caught, while `binding.pry` and `byebug` correctly detected.

I use a pretty useful debugging gem for running a single test from a suite, just stick a focus above a test (https://github.com/seattlerb/minitest-focus)

```ruby
focus
test 'only run this test' do
  # debugging something important
end
```

The downside is that these `focus` statements sometimes get accidentally left in commits, while `binding.pry` or `byebug` get picked up by rubocop

Would something like this belong in `RuboCop::Cop::Lint::Debugger`?

**Comment by bbatsov** [comment]
The problem is that this method name is more generic than all other debug expressions, so I'd rather not add it directly. Maybe we should make the cop configurable to some extent, so people can add additional expressions if they need to.

**Comment by backus** [comment]
This would be a nice addition to an equivalent `rubocop-minitest` extension similar to [`rubocop-rspec`](http://github.com/backus/rubocop-rspec). `rubocop-rspec` has a focused test cop: https://github.com/backus/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/focus.rb

**Comment by CyborgMaster** [comment]
I would like to see the debugger cop more configurable for the same reason.  We use the focus gem and would love to flag that in our test directory

**Comment by CyborgMaster** [comment]
For now, I have solved this in our codebase by writing our own very very simple cop.

``` ruby
module RuboCop::Cop::Ally
  class NoFocus < RuboCop::Cop::Cop
    MSG = 'Remove `focus` from tests.'

    def_node_matcher :focused?, <<-MATCHER
      (send nil? :focus)
    MATCHER

    def on_send(node)
      return unless focused?(node)

      add_offense node
    end
  end
end
```

**Comment by jpickwell** [comment]
Sorry, but just a quick aside: why is there a `Cop` module and a `Cop` class?!
